packages/Kokkos/src/details/DTK_DetailsTreeTraversal.hpp

```diff
@@ -92,14 +92,26 @@ KOKKOS_FUNCTION void spatial_query( BVH<NO> const bvh,
         {
             indices[pos] = BVHQuery<NO>::getIndex( bvh, node );
             ++pos;
+#if HAVE_DTK_DBC
+            if ( pos > n_max_indices )
+                printf( "Increase the size of indices array\n" );
```

Comment:
yeah but it would be super ugly for the user to understand what's going on. You have to run your code in `cuda-gdb` to see what's happening this is a lot simpler.